src/test/scala/com/cognite/spark/datasource/TimeSeriesRelationTest.scala

```diff
@@ -145,6 +147,7 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest {
            |select description,
            |isString,
            |name,
+           |name as legacyName,
```

Comment:
Remove this from tests, we don't want to change the external schema and it doesn't look like we are. :+1: